tests/h/views/main_test.py

```diff
@@ -39,6 +39,7 @@ def test_og_no_document(render_app_html, pyramid_request):
     assert any(test(d) for d in kwargs['extra']['meta_attrs'])
 
 
+@pytest.mark.usefixtures('render_app')
```

Comment:
This should just be one-line: `@pytest.mark.usefixtures('render_app', 'routes')`